mathesar_ui/src/component-library/dropdown/Dropdown.scss

```diff
@@ -22,6 +24,14 @@ button.dropdown.trigger {
     align-items: center;
     padding: 0 0.1rem;
   }
+  &:focus {
+    outline: 2px solid var(--slate-300);
+    outline-offset: 1px;
+    border-radius: var(--border-radius-s);
+  }
+  &:hover {
+    background: var(--slate-100);
+  }
```

Comment:
The hover state here will affect all types of dropdowns (which allow different appearances similar to buttons), and we cannot provide a common background color for all of them.

Refer the screenshot below for the 'Shortcuts' button. There are other places in the app where this issue might be present as well.

<img width="307" alt="Screenshot 2023-11-27 at 7 01 40 PM" src="https://github.com/mathesar-foundation/mathesar/assets/11850603/5a21b116-da32-4b85-96cd-3560c234d93a">

Reply:
I removed it from the global file and added Styles for `TableCard.svelte` so it wouldn't effect other things.